player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -634,6 +640,9 @@ public void onSkipClick() {
     public void onOpenClick() {
 
         VASTLog.v(TAG, "onOpenClick");
+        if (mBannerBitmap != null) {
+            mIsBannerSet = true;
+        }
```

Comment:
The reason of doing this, `load` calls `setEmptyState`. Moreover, downloading of banner is not part of Player, it's responsibility of caller.